build.gradle.kts

```diff
@@ -84,3 +85,21 @@ publishing {
         }
     }
 }
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        ktlint().editorConfigOverride(
+            mapOf(
+                "indent_size" to "4",
+                "disabled_rules" to "package-name"
+            )
+        )
+    }
+    kotlinGradle {
+        target("*.gradle.kts")
+        ktlint().editorConfigOverride(
+            mapOf("indent_size" to "4")
```

Comment:
Why do you set `indent_size`?

Reply:
I found spotless does not fix intendations in build.gradle without `set indent_size`

Reply:
Thank you for explanation.